pkg/cluster/cluster.go

```diff
@@ -345,6 +356,12 @@ func (os *ClusterSpec) WithDefaults() *ClusterSpec {
 	if s.AdditionalWalSenders == nil {
 		s.AdditionalWalSenders = Uint16P(DefaultAdditionalWalSenders)
 	}
+	if s.AdditionalSyncStandbyNames == nil {
+		s.AdditionalSyncStandbyNames = StringP(DefaultSyncStandbyNames)
+	}
+	if s.AdditionalReplicationSlotName == nil {
+		s.AdditionalReplicationSlotName = StringP(DefaultReplicationSlotName)
+	}
```

Comment:
After making them a `[]string` the values should be validated to check that they're valid sync standby names and replication slot names.

Reply:
AdditionalReplicationSlotName - is the name of a single replication slot, not an array

Reply:
yes, I was asking to make it an array so multiple additional replication slots can be defined like done for additionalSyncStandbyNames.

Reply:
Simon, in our task we need one definite replication slot. But if you insist we can extend the basic idea. Do you?

Reply:
@saranhada yes, since we are providing a new option to define an additional replication slotlets make it useful for everyone giving the ability to define N additional replication slots if needed.